mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -227,9 +227,10 @@ func Provider() *schema.Provider {
 			"google_compute_network":                           dataSourceGoogleComputeNetwork(),
 			"google_compute_network_endpoint_group":            dataSourceGoogleComputeNetworkEndpointGroup(),
 			"google_compute_node_types":                        dataSourceGoogleComputeNodeTypes(),
-			"google_compute_regions":                           dataSourceGoogleComputeRegions(),
+			"google_compute_region_network_endpoint_group":		dataSourceGoogleComputeRegionNetworkEndpointGroup(),
```

Comment:
I believe this list is roughly alphabetized, making this line appear below the next ("google_compute_region_instance_group")